### PR TITLE
Fix env vars and added custom connection impl

### PR DESCRIPTION
**Pull request by ritesh-modi**
Fix env vars related bug and added custom connection implementation

### CODE REVIEW

File: named_entity_recognition/configs/deployment_config.json [hunk 1]
```diff
@@ -3,12 +3,12 @@
         {
             "ENV_NAME": "dev",
             "TEST_FILE_PATH": "sample-request.json",
-            "ENDPOINT_NAME": "",
+            "ENDPOINT_NAME": "gfdgdfgdg",
```
Comment (mohanajuhi166) [inline named_entity_recognition/configs/deployment_config.json:6]: just checking if this is required?

File: named_entity_recognition/configs/deployment_config.json [hunk 2]
```diff
@@ -3,12 +3,12 @@
...
             "ENDPOINT_DESC": "An online endpoint serving a flow for named entity flow",
             "DEPLOYMENT_DESC": "prompt flow deployment",
             "PRIOR_DEPLOYMENT_NAME": "",
             "PRIOR_DEPLOYMENT_TRAFFIC_ALLOCATION": "",
-            "CURRENT_DEPLOYMENT_NAME": "",
+            "CURRENT_DEPLOYMENT_NAME": "fdgdgdg",
```
Comment (mohanajuhi166) [inline named_entity_recognition/configs/deployment_config.json:11]: same as above

File: named_entity_recognition/configs/deployment_config.json (hunk 1 above)
Comment (ritesh-modi) [reply named_entity_recognition/configs/deployment_config.json]: removed them

File: named_entity_recognition/configs/deployment_config.json (hunk 2 above)
Comment (ritesh-modi) [reply named_entity_recognition/configs/deployment_config.json]: removed it

**Review by mohanajuhi166 — APPROVED** [review]
LGTM

**Pull request merged**